src/usps_processor/api.clj

```diff
@@ -38,11 +38,15 @@
             (Integer/parseInt)
             zipcode->city-state)))
 
-(defn render-scan [scan]
-  (-> scan
-      (select-keys [:scan/time :scan/barcode :scan/facility-zip
-                    :scan/operation-code :scan/service])
-      attach-facility-city-state))
+(defn render-scan
+  [scan & [attach-mailing]]
+  (let [scan-render (-> scan
+                        (select-keys [:scan/time :scan/barcode :scan/facility-zip
+                                      :scan/operation-code :scan/service])
+                        attach-facility-city-state)]
+    (if attach-mailing
```

Comment:
Actually Troy and I ended up refactoring this such that render-scan is unchanged from kraken (and master too, IIRC) and we merge in the mailing in `queue/publish-scan` instead, which feels cleaner. More commits coming soon.